### DISCUSSION TITLE
doris 写入动态分区表1000w条数据30min ，如何优化

### DISCUSSION THREAD

**Discussion by shuimanlinling** [apache/doris#17340] [A - General / Q&A]
主要参数：    option("doris.deserialize.arrow.async",'true').\
    option('doris.sink.task.partition.size',100).\
    option('doris.sink.task.use.repartition', 'true').\

**Comment by zixuedanxin** [comment]
我也遇到同样的问题，即使是spark-connector 数据批量导入最好也就8000条/s

**Comment by hwanginsitein** [comment]
doris.sink.task.use.repartition 这个参数经常要设置成true的吗？
